### ISSUE TITLE
[coordinate-service] no root breaks build (Trac #1121)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1115]
```text
-- + coordinate-service
CMake Error at cmake/tools.cmake:83 (message):
  Attempt to use tool 'root' which wasn't found
Call Stack (most recent call first):
  cmake/tools.cmake:132 (use_tool)
  cmake/project.cmake:223 (use_tools)
  coordinate-service/CMakeLists.txt:23 (i3_add_library)
```

Not having root should just disable the project, not break the build.

Migrated from https://code.icecube.wisc.edu/ticket/1121
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "{{{\n-- + coordinate-service\nCMake Error at cmake/tools.cmake:83 (message):\n  Attempt to use tool 'root' which wasn't found\nCall Stack (most recent call first):\n  cmake/tools.cmake:132 (use_tool)\n  cmake/project.cmake:223 (use_tools)\n  coordinate-service/CMakeLists.txt:23 (i3_add_library)\n}}}\n\nNot having root should just disable the project, not break the build.", 
    "reporter": "david.schultz", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[coordinate-service] no root breaks build", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T15:42:43", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150817T15:44:17: nega commented:

> unless something's changed in the past 10 days, this isn't so.

**Comment by icecube-trac** [comment]
Trac update at 20150817T16:16:39: david.schultz commented:

> I think this is the result of removing FORCE from 
> 
> ```text
> set(USE_ROOT OFF CACHE BOOL "Root not detected")
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150817T16:36:49: 
* nega changed _comment0 from:

> The following all exhibit the same behavior. They should be changed to use ROOT_FOUND, or the `cmake/tools/root.cmake` should set USE_ROOT to false if ROOT_FOUND is also false.
> {{{ 
> ../src/AtmCscdEnergyReco/CMakeLists.txt
> ../src/coordinate-service/CMakeLists.txt
> ../src/cscd-llh/CMakeLists.txt
> ../src/dataclasses/CMakeLists.txt
> ../src/dst-extractor/CMakeLists.txt
> ../src/examples/CMakeLists.txt
> ../src/filterscripts/CMakeLists.txt
> ../src/icetray/CMakeLists.txt
> ../src/lilliput/CMakeLists.txt
> ../src/ophelia/CMakeLists.txt
> ../src/phys-services/CMakeLists.txt
> ../src/rootwriter/CMakeLists.txt
> ../src/toprec/CMakeLists.txt
> ../src/topsimulator/CMakeLists.txt
> }}}
> 

to:

> 1439829462367943


* nega commented:

> The following all exhibit the same behavior. They should be changed to use ROOT_FOUND, or the `cmake/tools/root.cmake` should set USE_ROOT to false if ROOT_FOUND is also false. (I prefer the first option.)
> ``` 
> ../src/AtmCscdEnergyReco/CMakeLists.txt
> ../src/coordinate-service/CMakeLists.txt
> ../src/cscd-llh/CMakeLists.txt
> ../src/dataclasses/CMakeLists.txt
> ../src/dst-extractor/CMakeLists.txt
> ../src/examples/CMakeLists.txt
> ../src/filterscripts/CMakeLists.txt
> ../src/icetray/CMakeLists.txt
> ../src/lilliput/CMakeLists.txt
> ../src/ophelia/CMakeLists.txt
> ../src/phys-services/CMakeLists.txt
> ../src/rootwriter/CMakeLists.txt
> ../src/toprec/CMakeLists.txt
> ../src/topsimulator/CMakeLists.txt
> ```
>

**Comment by icecube-trac** [comment]
Trac update at 20150817T17:19:17: 
* nega changed resolution from "" to "fixed"
* nega changed status from "new" to "closed"
* nega commented:

> In changeset 136305:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136305"
> use ROOT_FOUND instead of USE_ROOT
> 
> USE_ROOT is a configuration option
> ROOT_FOUND is a configuration result
> 
> with r2221/IceTray, closes #1115
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
